tools/mo/requirements_tf2.txt

```diff
@@ -1,5 +1,6 @@
 tensorflow~=2.5.0
-networkx~=2.5
+networkx~=2.5; python_version <= "3.6"
+networkx~=2.6; python_version > "3.6"
```

Comment:
We also have other places still using 2.5 openvino\tools\openvino_dev\requirements.txt (and thirdparty/open-model-zoo).
@slyubimt, could you please recommend the correct way to change networkx version so we willn't affect other componenets

Reply:
Currently `openvino-dev` packaging script obtains requirements from setup.py of individual component. 
This change needs to be agreed  with components owners and aligned for components to void potential conflicts:
 - POT @AlexKoff88 :  https://github.com/openvinotoolkit/openvino/blob/5f53e6f697038d454f29777c81afe05020cdbec1/tools/pot/setup.py#L52

Reply:
I update NetworkX dependency in POT to align it with MO dependency.

@AlexKoff88 @KodiaqQ Are you ok with that?

Reply:
POT requirements list format was aligned with https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#platform-specific-dependencies

Reply:
POT CI check is green. MO conversion checked with NetworkX 2.6 and found no issues.